src/sections/Sponsors.vue

```diff
@@ -7,13 +7,15 @@
       <div class="tile sponsor-columns">
         <div v-for="(col, i) in columns" :key="'column-'+i" class="tile is-vertical sponsor-column">
           <div v-for="(s, j) in col" :key="'row-'+i+'-'+j" class="tile sponsor-container">
-            <img ref="sponsor-logo"
-              :src="s.logoURL"
-              :alt="s.name"
-              :style="{
-                filter: s.logoFilter,
-                'object-fit': 'contain',
-              }" />
+            <a :href="s.website" target="_blank">
```

Comment:
unfortunately the additional nesting has changed the alignment of the logos ( ͡° ʖ̯ ͡°) - before:

<img width="1590" alt="image" src="https://user-images.githubusercontent.com/23356519/79835769-1d506e00-8364-11ea-81c2-17f2df93bf07.png">

after:

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/23356519/79835722-0a3d9e00-8364-11ea-9b92-8f334e29cee9.png">

We might need to put `object-fit` on the `<a>`, or something like that